js/libs/keycloak-admin-client/src/defs/realmRepresentation.ts

```diff
@@ -74,6 +74,7 @@ export default interface RealmRepresentation {
   maxDeltaTimeSeconds?: number;
   maxFailureWaitSeconds?: number;
   maxTemporaryLockouts?: number;
+  bruteForceStrategy?: string;
```

Comment:
If `MULTIPLE` and `LINEAR` are the only options this can be more strictly typed as:
```suggestion
  bruteForceStrategy?: 'MULTIPLE' | 'LINEAR';
```